service/step_defs_test.go

```diff
@@ -3407,18 +3407,19 @@ func (f *feature) iSetNoProbeOnStart(value string) error {
 	return nil
 }
 
-func (f *feature) iCallGetSnapshotNameFromIsiPathWith(isiPath string) error {
+// shefali
+func (f *feature) iCallGetSnapshotNameFromIsiPathWith(exportPath string) error {
 	clusterConfig := f.service.getIsilonClusterConfig(clusterName1)
 	ctx, _, _ := GetRunIDLog(context.Background())
-	_, f.err = clusterConfig.isiSvc.GetSnapshotNameFromIsiPath(ctx, isiPath)
+	_, f.err = clusterConfig.isiSvc.GetSnapshotNameFromIsiPath(ctx, exportPath, "System", "/ifs")
```

Comment:
what if the "system" and "ifs" values gets changes? it will lead to the code change. can we pass it from the values or config file?

Reply:
For now I have hardcoded the zone name to System as snapshot test cases are for system zone and system zone base path is /ifs . will be updating this while fixing UTs in repo